APE1.rst

```diff
@@ -1,11 +1,11 @@
 APE Purpose and Process
 -----------------------
 
-author: Perry Greenfield
+author: Perry Greenfield, Lia Corrales
 
 date-created: 2013 October 11
 
-date-last-revised: 2013 November 8
+date-last-revised: 2020 July 22
```

Comment:
```suggestion
date-last-revised: 2020 July 22

revised-by: Lia Corrales 2020 July 22
```
and future revisions would add *additional* `revised-by` sections to follow https://github.com/astropy/astropy-APEs/pull/60#issuecomment-705783107

Reply:
Done - had to make change manually due to conflicts